behavior.h

```diff
@@ -44,13 +44,13 @@ void behavior_start(int b);
 void behavior_stop(int b);
 int behavior_enabled(int b);
 
-#define B_ALL 			0xFFFF
-#define B_SOUND_BUTTON 		(1 << 0)
-#define B_LEDS_BUTTON 		(1 << 1)
+#define B_ALL			0xFFFF
+#define B_SOUND_BUTTON	(1 << 0)
```

Comment:
my editor was first filling tabs with space. I remove this options afterwards and run the format tools. But it was not align in my view. I can remove this changes or not...